src/common/modal.js

```diff
@@ -26,7 +26,7 @@ export const alert = (title, body) => {
         <Header closeButton>
           <Title>{title}</Title>
         </Header>
-        <Body>{body}</Body>
+        {body && <Body>{body}</Body>}
```

Comment:
I am not sure it's good design, and the Bootstrap component does not seem to be intended to be used without a body.

You could simply add a text like: *The remote is reachable and appears to be working correctly.*